mne/simulation/tests/test_source.py

```diff
@@ -237,10 +236,9 @@ def test_simulate_sparse_stc_single_hemi(_get_fwd_labels):
     assert_array_equal(stc_1.rh_vertno, stc_2.rh_vertno)
 
     # smoke test for new API
-    if check_version('numpy', '1.17'):
-        simulate_sparse_stc(fwd['src'], len(labels_single_hemi), times,
-                            labels=labels_single_hemi,
-                            random_state=np.random.default_rng(0))
+    simulate_sparse_stc(fwd['src'], len(labels_single_hemi), times,
+                        labels=labels_single_hemi,
+                        random_state=np.random.default_rng(0))
```

Comment:
if this was a smoke test for the new numpy rng api, do we even need it at all anymore?

Reply:
I think it's good to have tests for check_random_state that the new API can be used, I guess we could keep those just in `utils/` though